### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by blackinno**
- Default Button
- Button primary type submit
- Button secondary with href with an external link
- Button secondary with href with an internal link
- Button white
- Button outline
- Tag a with an icon

### CODE REVIEW

File: components/Button/Button.vue [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        return 'far fa-external-link'
+      }
+      return 'fa fa-arrow-right'
+    }
+  }
+}
+</script>
+
+<style scoped>
+@import '../../assets/style.css';
```
Comment (antho1404) [inline components/Button/Button.vue]: this should not be included in the component.

Either the application calling it needs to define it (or use this default one) or you can create a Theme component that actually style this.

File: components/Button/Button.stories.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    components: { Button },
+    template: `<Button white>TEST</Button>`
+  }))
+  .add('Outline', () => ({
+    components: { Button },
+    template: `<Button outline>TEST</Button>`
+  }))
+  .add('Tag a with icon', () => ({
+    components: { Button },
+    template: `<a href='https://github.com/mesg-foundation' target="_blank" ><i class="fab fa-github"></i> Github MESG Foundation</a>`
```
Comment (antho1404) [inline components/Button/Button.stories.js]: On the storybook this one is not designed https://deploy-preview-7--mesg-ui.netlify.com/?path=/story/button--tag-a-with-icon

**Review by antho1404 — CHANGES_REQUESTED** [review]

File: components/Button/Button.vue (hunk 1 above)
Comment (blackinno) [reply components/Button/Button.vue]: I will try to create a default theme

File: components/Button/Button.stories.js (hunk 2 above)
Comment (blackinno) [reply components/Button/Button.stories.js]: Remove, it's done.

**Comment by antho1404** [comment]
The button looks bigger than the one on the website. Is that normal?

**Comment by blackinno** [comment]
I think about the storybook display because I use button stylesheet from mesg.com.

**Review by antho1404 — APPROVED** [review]
ok it's fine, you are displaying a button primary and not a link primary. The link has the right size

**Pull request merged**